snmp/datadog_checks/snmp/data/default_profiles/barracuda-cloudgen.yaml

```diff
@@ -0,0 +1,206 @@
+extends:
+  - barracuda.yaml
+
+sysobjectid:
+  - 1.3.6.1.4.1.10704.1.*
+
+metrics:
+  - MIB: PHION-MIB
+    table:
+      OID: 1.3.6.1.4.1.10704.1.0
+      name: boxServices
+    symbols:
+      - name: boxServices
```

Comment:
I'm wondering if we should prefix all metrics in this file with `snmp.phion.` for example
since the metric names are very short

Example: `snmp.phion.boxServices`

That can help avoid future conflicts.